pkgs/tools/text/papeer/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "papeer";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "lapwat";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-N0ZOVP64+2iyR04GSgwIGQZ5VSgpwGrsN7+quyob9iA=";
+  };
+
+  vendorHash = "sha256-M9fe0hHC/u4b36V7Ux/Z/hBgZjYS0Ve50ftPLT4CiAg=";
+
+  doCheck = false; # uses network
+
+  meta = src.meta // {
+    description = "Convert websites into ebooks and markdown";
+    license = lib.licenses.free;
```

Comment:
What license, precisely?

Reply:
Unspecified.

Reply:
I'm not a lawyer, but "unspecified license" defaults to nonfree. I will ask them.

Reply:
If you aren't a lawyer then keep silent. They published code without licensing conditions, that makes it free.

Reply:
> If you aren't a lawyer then keep silent.

Not being a lawyer does not imply either not being affected by laws, nor knowing a thing or other about them.

Some time ago I went through a similar situation: a software without explicit license. I reached the upstream asking for a license and they put a simple explicit MIT one.

As a precaution I decided to manifest this concern.

> They published code without licensing conditions, that makes it free.

This is not how copyright works. A quick Google returns:

https://www.synopsys.com/blogs/software-security/unlicensed-open-source-scenarios/

> In the U.S. and many other places, creative work (including software) is protected by **exclusive copyright by default**. This means that no one can legally use, copy, distribute, or modify that software without explicit permission from the creator/author. This permission comes in the form of a license that grants the right to do so. Without that license, **the baseline assumption is that you do not have permission to use the software**.

Again, I am not a lawyer. However I will not stay silent when there are plausible reasons of concern.

Reply:
The punishment for a non-lawyer doing a lawyer's work in the US can be assumed to be worse than the outcomes of acting in ignorance.

Reply:
Are you a lawyer?